.github/workflows/container.yml

```diff
@@ -82,3 +82,32 @@ jobs:
           platforms: linux/amd64,linux/arm64
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+      - name: 'Setup meta information (IS_VERSION_TAG: ${{ env.IS_VERSION_TAG }}, IS_LATEST_TAG: ${{ env.IS_LATEST_TAG }} )'
+        id: meta-oldstable
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ github.repository }}
+          labels: |
+            org.opencontainers.image.vendor=Greenbone
+            org.opencontainers.image.base.name=debian:oldstable-slim
+          flavor: latest=false # no auto latest container tag for git tags
+          tags: |
+            # for the images provided for debian:oldstable we just provide
+            # oldstable on an new version or oldstable-edge when it is on main.
+            # oldstable-branch-sha on a branch
+            type=raw,value=stable,enable=${{ env.IS_LATEST_TAG }}
```

Comment:
I think the name for the image should ne oldstable rather than stable for the oldstable image?

Reply:
oh, I forgot to change that one. thanks